glue/core/visual.py

```diff
@@ -15,20 +15,29 @@
 class VisualAttributes(HasCallbackProperties):
 
     """
-    This class is used to define visual attributes for any kind of objects
-
-    The essential attributes of a VisualAttributes instance are:
-
-    :param color: A matplotlib color string
-    :param alpha: Opacity (0-1)
-    :param linewidth: The linewidth (float or int)
-    :param linestyle: The linestyle (``'solid' | 'dashed' | 'dash-dot' | 'dotted' | 'none'``)
-    :param marker: The matplotlib marker shape (``'o' | 's' | '^' | etc``)
-    :param markersize: The size of the marker (int)
-
+    This class is used to define visual attributes for any kind of objects.
+
+    Parameters
+    ----------
+    parent : :class:`~PyQt5.api.qtcore.qobject`, optional
```

Comment:
```suggestion
    parent : `QObject`, optional
```
I'll try to get this sorted out in a separate numpydoc PR!